pkg/diff/diff_test.go

```diff
@@ -786,15 +785,13 @@ func TestStructuredMergeDiff(t *testing.T) {
 		require.NoError(t, err)
 		assert.NotNil(t, result)
 		assert.True(t, result.Modified)
-		assert.Equal(t, string(expectedLiveBytes), string(result.PredictedLive))
+		svc := YamlToSvc(t, result.PredictedLive)
+		assert.Len(t, svc.Spec.Ports, 2)
```

Comment:
Why can we no longer do a full text comparison?

Reply:
Basically because it is terrible to maintain. I also found it hard to reason about what the test is actually validating. This way is much cleaner and easier to maintain.